05.ls/ls.rb

```diff
@@ -35,89 +38,74 @@ def output_no_option
   puts formatted_list.transpose.map(&:join)
 end
 
-def print_file_list(file_stats)
+def print_files(file_stats)
   hard_link_padding, user_padding, group_padding, file_size_padding = calculate_padding(file_stats)
-  files_list = make_file_stat_list(file_stats)
-  files_list.each do |file_list|
-    print file_list[:permission].ljust(10, ' ')
-    print file_list[:hard_link].rjust(hard_link_padding, ' ')
-    print file_list[:user].rjust(user_padding, ' ')
-    print file_list[:group].rjust(group_padding, ' ')
-    print file_list[:file_size].rjust(file_size_padding, ' ')
-    print file_list[:time]
-    puts file_list[:file_name]
+  total_block_size = file_stats.sum { |block_size| block_size[:blocks_size] }
+  puts("total #{total_block_size}")
```

Comment:
```suggestion
  puts "total #{total_block_size}"
```

他のputsやprintには()が付いてないので。